core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -90,6 +90,7 @@ oppia.directive('audioTranslationBar', [
           $scope.showRecorderWarning = false;
           $scope.audioLoadingIndicatorIsShown = false;
           $scope.checkingMicrophonePermission = false;
+          $scope.isAudioTimerShown = true;
```

Comment:
Try naming boolean variables as: `$scope.audioTimerIsShown`.

For function names, is/are is used as starting since verbs distinguish them from variables. You can also refer [this](https://github.com/oppia/oppia/pull/5997#discussion_r247320899) comment

Reply:
@ankita240796 Kindly see [this](https://github.com/oppia/oppia/pull/5986#discussion_r246992691)
Thanks! :)

Reply:
Hi @rakshitkumarcse, I just took a look and made a correction to that comment! Thanks for pointing it out :)

Reply:
Welcome @seanlip :)

Reply:
The change is made. Thanks! @ankita240796 @seanlip